src/Engine/ProtoCore/Core.cs

```diff
@@ -214,7 +214,13 @@ public struct ErrorEntry
         public List<CodeBlock> CodeBlockList { get; set; }
         // The Complete Code Block list contains all the code blocks
         // unlike the codeblocklist which only stores the outer most code blocks
-        public List<CodeBlock> CompleteCodeBlockList { get; set; }
+        [Obsolete("Property will be deprecated in Dynamo 3.0")]
+        public List<CodeBlock> CompleteCodeBlockList
+        {
+            get { return CompleteCodeBlockDict.Select(x => x.Value).ToList(); }
+            set { value.ForEach(x => CompleteCodeBlockDict.Add(x.codeBlockId, x)); }
```

Comment:
What is value here, a dictionary or list?

Reply:
value represents the `CompleteCodeBlockList`
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/value